app/controllers/nonsipp/loansmadeoroutstanding/LoansListController.scala

```diff
@@ -338,7 +338,7 @@ object LoansListController {
         List()
       case (list, _) =>
         list.flatMap { case LoansData(index, loanAmount, recipientName, loansProgress, canRemove) =>
-          val validLoan = loanAmount.value > 0 && recipientName.nonEmpty
+          val validLoan = recipientName.nonEmpty
```

Comment:
Because of the `loanAmount.value > 0` removal and `validLoan`'s usage further down the line, e.g. in 
  `case _ if loansProgress.inProgress && !isPrePopulation => None`
  I suggest to do some extra testing of the rendering of this list page. E.g. in pre-population mode or anywhere else the display logic can be affected.